bin/desi_daily_archive.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+# Licensed under a 3-clause BSD style license - see LICENSE.rst.
+#
+if [[ -z "${DESIDA}" ]]; then
+    echo "ERROR: DESIDA is undefined!"
+    exit 1
+fi
+source ${DESIDA}/bin/desida_library.sh
+#
+# Help message.
+#
+function usage() {
+    local execName=$(basename $0)
+    (
+    echo "${execName} [-B] [-h] [-j JOBS] [-s DIR] [-V] [-v]"
+    echo ""
+    echo "Generate checksum and backup jobs for spectro/redux/daily/tiles/archive."
+    echo ""
+    echo "    -B         = If set, do NOT submit batch jobs."
+    echo "    -h         = Print this message and exit."
+    echo "    -j JOBS    = Use JOBS directory to write batch files (default ${DESI_ROOT}/users/${USER}/jobs)."
+    echo "    -s DIR     = Use DIR for temporary files (default ${SCRATCH})."
+    echo "    -V         = Version. Print a version string and exit."
+    echo "    -v         = Verbose mode. Print extra information."
+    echo ""
+    ) >&2
+}
+#
+# Create jobs.
+#
+function create_archivedate_job() {
+    local tileid=$1
+    local archivedate=$2
+    local logs=$3
+    local job_name="redux_daily_tiles_archive_${tileid}_${archivedate}"
+    cat > ${jobs}/${job_name}.sh <<EOT
+#!/bin/bash
+#SBATCH --account=desi
+#SBATCH --qos=xfer
+#SBATCH --constraint=cron
+#SBATCH --time=30:00
+#SBATCH --job-name=${job_name}
+#SBATCH --output=${jobs}/%x-%j.log
+#SBATCH --licenses=cfs,scratch
```

Comment:
Is there also a licenses option to ensure that HPSS is available when this job runs?